packages/core/src/js/integrations/reactnativeerrorhandlers.ts

```diff
@@ -155,7 +155,8 @@ function setupErrorUtilsGlobalHandler(): void {
       return;
     }
 
-    void client.flush(client.getOptions().shutdownTimeout || 2000).then(
+    // shutdownTimeout was removed: https://github.com/getsentry/sentry-javascript/pull/15217
```

Comment:
l: We can probably also remove `shutdownTimeout` from [RNSentryModuleImpl.java](https://github.com/getsentry/sentry-react-native/pull/4566/files#diff-fe8d5afb81105308bdadba53d94f67ee2b7142f0f003d85a44dcd8d65ee2e2e5)

Reply:
Nice catch! fixed on https://github.com/getsentry/sentry-react-native/pull/4568/commits/9a0b21ecf6b7f52098035b36837d41296502532f

Reply:
I think we should move the timeout type to RN since the native SDKs still use it.

https://github.com/getsentry/sentry-react-native/pull/4568/files#r1992043223